install.sh

```diff
@@ -262,6 +266,59 @@ outdated_glibc() {
   version_lt "$glibc_version" "$REQUIRED_GLIBC_VERSION"
 }
 
+usage() {
+  cat <<EOS
+Homebrew Installer
+Usage: /bin/bash install.sh [--help] [--brew-git-remote [URL]] [--core-git-remote [URL]]
+
+This script installs Homebrew, the missing package manager for macOS (or Linux).
+
+Options:
+  -b, --brew-git-remote=URL   Use this URL as the Homebrew/brew git remote.
+                              If no argument is given, the default remote is used.
+                              Default: ${tty_underline}${HOMEBREW_BREW_DEFAULT_GIT_REMOTE}${tty_reset}.
+  -c, --core-git-remote=URL   Use this URL as the Homebrew/homebrew-core git remote.
+                              If no argument is given, the default remote is used.
+                              Default: ${tty_underline}${HOMEBREW_CORE_DEFAULT_GIT_REMOTE}${tty_reset}.
+  -h, --help                  Show this help message and exit.
+EOS
+  exit "${1:-0}"
+}
+
+# Parse remote URLs of Homebrew repositories from command line arguments if presented.
+unset HOMEBREW_{BREW,CORE}_GIT_REMOTE  # unset variables from environment at first
```

Comment:
We can keep these variables from environments if remove the `unset` statement here. The priority order for `HOMEBREW_{BREW,CORE}_GIT_REMOTE` is:
1. from arguments if specified
2. from environment if set
3. the default value

But I think it could be difficult to write the usage clear enough for users.